lib/utils/__tests__/checkAgainstRule.test.mjs

```diff
@@ -26,6 +26,8 @@ const mockResult = {
 				},
 			},
 		},
+		ruleSeverities: {},
+		fixersData: {},
```

Comment:
question

That means that

```
		customMessages: {},
		customUrls: {},
		ruleMetadata: {},
		disabledRanges: {},
```
are optional?

Reply:
No, they are not optional from the type definition:
https://github.com/stylelint/stylelint/blob/2dfe7cea14159b536b83a35ab9423cd643b36a7e/types/stylelint/index.d.ts#L156-L169

I pushed 09a724ee3 to clarify that. Any thoughts?